specification/pom.xml

```diff
@@ -205,44 +205,6 @@
                             </attributes>
                         </configuration>
                     </execution>
-                    <!-- ManagedBeans HTML generation -->
```

Comment:
I assume this is for moving managed beans to the core profile.

Reply:
The managed beans spec was moved to its own repo:
https://github.com/eclipse-ee4j/managed-beans-spec

It was deprecated for EE10 as I remember, so I'm not sure what we are doing with that spec.